azext_iot/tests/test_iot_central_int.py

```diff
@@ -197,7 +197,7 @@ def test_device_connect(self):
         device_id = "testDevice"
 
         device_primary_key = self.cmd(
-            "iot central device compute-device-key create --pk {} -d {}".format(
+            "iot central device compute-device-key --pk {} -d {}".format(
```

Comment:
@valluriraj does this align with PM ask?

Reply:
synced up with PM good to go.